guides/continuous-integration.md

```diff
@@ -1,5 +1,24 @@
-# CI/CD
+# Continuous integration & delivery
 
-CI/CD
+CI/CD is used across all projects at the Data Shed as a way of streamlining
```

Comment:
This is the first time the acronyms "CI" and "CD" are introduced, and as such I'd recommend they are explained in full 😊